third_party/terraform/tests/resource_dataproc_cluster_test.go

```diff
@@ -870,7 +870,7 @@ resource "google_dataproc_cluster" "with_config_overrides" {
 			machine_type      = "n1-standard-1"
 			disk_config {
 				boot_disk_type    = "pd-standard"
-				boot_disk_size_gb = 11
```

Comment:
I think this `11` and `12` are meant to be distinct values on purpose. Can you update them to `16` and `17`?